### PR TITLE
No longer support images for annual reports

### PR DESCRIPTION
**Pull request by nlisgo**
Images are currently required for annual reports but we are not displaying them on journal.

See: https://github.com/elifesciences/jira-import/issues/4192

### CODE REVIEW

File: src/model/annual-report.v2.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        type: string
+        format: uri
+    title:
+        description: Title
+        type: string
+        minLength: 1
+    impactStatement:
+        description: Description of the annual report
+        type: string
+    image:
```
Comment (thewilkybarkid) [inline src/model/annual-report.v2.yaml]: Could drop it completely? Don't think it's used anywhere anymore...

Comment (nlisgo) [reply src/model/annual-report.v2.yaml]: I agree. I'll make that adjustment.

**Comment by nlisgo** [comment]
@thewilkybarkid are you happy for this to be deployed? I've marked this new version as experimental until we actually do the work on `journal-cms`.

**Review by thewilkybarkid — APPROVED** [review]

**Comment by giorgiosironi** [comment]
Merging this to be able to build correctly the downstream PRs.

**Pull request merged**